sg_jira/shotgun_session.py

```diff
@@ -64,7 +64,9 @@ def __init__(self, base_url, script_name=None, *args, **kwargs):
             **safe_kwargs
         )
 
-        self._shotgun_schemas = {}
+        self._shotgun_entities = []  # will be used to store FPT entities list
```

Comment:
I think this might be better renamed as `_shotgun_entity_types`, initially I thought it was going to be a list of the entity records.